### ISSUE TITLE
Sass compile fails with #{ } constructions

### ISSUE THREAD

**Issue by c1aphas** [sass/node-sass#1232]
Hi!
Problem code:
```
foo {
  font: 22px/#{28/22} bar
}
```

It works fine if i do:

```
foo {
  line-height: #{28/22}
}
```

But fails when inline.

```
$ nodejs --version
v4.2.1
$ npm --version
3.3.10
```

On osX error:

```
[1]    18135 bus error
```

On Ubuntu:

```
Segmentation fault (core dumped)
```

**Comment by xzyfer** [comment]
This is fixed in the latest libsass and will be in the next node-sass version.

**Comment by xzyfer** [comment]
This is fixed in the upcoming 3.5.0-beta. Subscribe to #1349 for further updates.
